src/components/molecules/CheckboxField/__snapshots__/CheckboxField.test.tsx.snap

```diff
@@ -110,6 +111,11 @@ exports[`<CheckboxField /> renders the component with no control icon and no a11
   box-shadow: 0 0 4px 0 #009E8E;
 }
 
+.c1:hover:checked:not(:disabled) + label:before,
+.c1:focus:checked + label:before {
+  background-color: #FFFFFF;
```

Comment:
And this one?

Reply:
does it make more sense to have either both transparent or both white so theyre consistent